runtime/wkt/src/google/protobuf/any.proto

```diff
@@ -0,0 +1,159 @@
+// DOT NOT EDIT
+// This file is synced from remote repo:
+// git@github.com:protocolbuffers/protobuf.git/src/google/protobuf/any.proto@sha=7bb8b108d16252d0ed053673d70ea6d2020ec7ff
```

Comment:
Did these files get added to this PR by mistake? I saw you had some changes related to this in your other PR, so I'm guessing these should be included there.

Reply:
yeah you are right. these don't belong here. I don't think they need to be committed in the other PR either. Committing them does however save from needing to do it during the build, which might not be a significant gain given the project size is not large.